util/chplenv/chpl_llvm.py

```diff
@@ -418,6 +427,34 @@ def is_system_clang_version_ok(clang_command):
 # use. Returns '' if no acceptable system clang was found.
 @memoize
 def get_system_llvm_clang(lang):
+    lang_upper = lang.upper()
+    if lang_upper == 'C++':
+        lang_upper = 'CXX'
+
+    # compute it based on setting CHPL_LLVM_CLANG_C/CXX
+    # or, if CHPL_TARGET_COMPILER=llvm, CHPL_TARGET_CC/CXX
+    tgt_llvm = overrides.get('CHPL_TARGET_COMPILER', 'llvm') == 'llvm'
+    if lang_upper == 'C':
+        llvm_clang_c = overrides.get('CHPL_LLVM_CLANG_C', '')
+        if llvm_clang_c != '':
```

Comment:
Is it possible that `clang_c` could be `None` here rather than the empty string?

Reply:
I don't think so, because the second argument to `overrides.get` is the value to produce if it's not found.